### ISSUE TITLE
Broken type inference with `import` ... `as`

### ISSUE THREAD

**Issue by cpboyd** [rrousselGit/freezed#494]
**Describe the bug**
I added
```dart
import 'package:quiver/strings.dart' as qs;
```
to my class and in the `freezed` generated code, it prepended `qs.` to all of the properties' types even though there are no types exported by that Quiver package.

**To Reproduce**
```dart
import 'package:freezed_annotation/freezed_annotation.dart';
import 'package:quiver/strings.dart' as qs;

part 'fb_account.freezed.dart';
part 'fb_account.g.dart';

@freezed
class FbAccount with _$FbAccount {
  const FbAccount._(); // Private constructor required for methods
  const factory FbAccount({
    @Default('') String name,
    @deprecated String? id,
    String? uri,
  }) = _FbAccount;

  String? get link {
    if (uri != null) return uri;
    return (qs.isEmpty(id))
        ? null
        : 'profile?id=${id!}';
  }

  factory FbAccount.fromJson(Map<String, dynamic> json) =>
      _$FbAccountFromJson(json);
}
```

**Expected behavior**
Type inference should not be impacted when using `import` ... `as`

**Comment by cpboyd** [comment]
A workaround was to simply use 
```dart
import 'package:quiver/strings.dart' show isEmpty;
```
but I like to import it as `qs` to more easily identify where I'm using quiver functions.

**Comment by cpboyd** [comment]
Possibly a regression from `0.14.3` to `0.14.4` as that's the only package that changed when this stopped working as expected.

**Comment by agrapine** [comment]
> Possibly a regression from `0.14.3` to `0.14.4` as that's the only package that changed when this stopped working as expected.

indeed. which is funny as the 0.14.4 was supposed to add support for `import ... as`  :smile:

**Comment by rrousselGit** [comment]
Eh, I forgot to check that case 🤷 

Let me quick-fix that

**Comment by rrousselGit** [comment]
Fixed as part of v0.14.5
